src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -64,6 +64,7 @@ export default (accessToken, userID, requestIDs) => {
       "is_saved"
     ),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
+    saleArtworksAllLoader: gravityLoader("sale_artworks"),
```

Comment:
@ashkan18 - this is where I defined the new loader endpoint; and the line just below is the one being used in the current query and elsewhere.